src/libs/EmojiUtils.ts

```diff
@@ -569,6 +569,120 @@ function getSpacersIndexes(allEmojis: EmojiPickerList): number[] {
     return spacersIndexes;
 }
 
+/**
+ * Split string into plain text and emojis array with attention to:
+ * Surrogate pairs (combined emojis including flags)
+ * Modifiers (different skin tones and emoji variations)
+ * @param text
+ */
+
+// Surrogate pairs (combined emojis)
+const highSurrogateStart = 0xd800;
```

Comment:
Let's use `SCREAMING_SNAKE_CASE` for all these constants

Reply:
sure! updated